VideoLocker/res/layout/row_discussion_user_profile.xml

```diff
@@ -13,44 +13,37 @@
         android:id="@+id/profile_image"
         android:layout_width="@dimen/discussion_image_diameter"
         android:layout_height="@dimen/discussion_image_diameter"
+        android:layout_gravity="center_vertical"
         android:layout_marginEnd="@dimen/discussion_responses_standard_margin"
         android:layout_marginRight="@dimen/discussion_responses_standard_margin"
         android:src="@drawable/profile_photo_placeholder" />
 
     <LinearLayout
         android:layout_width="wrap_content"
-        android:layout_height="match_parent"
+        android:layout_height="wrap_content"
         android:gravity="center_vertical"
```

Comment:
By setting the layout height to `wrap_content`, you've rendered the vertical gravity meaningless. You should probably just set the gravity as vertically centered in the parent layout, and remove the `gravity`/`layout_gravity` attributes from the children.